### DISCUSSION TITLE
display question

### DISCUSSION THREAD

**Discussion by montyfert** [arendst/Tasmota#20017] [Q&A] [answered]
hi i have a display and dht11 connected to a d1 mini i have the temperature displayed on the screen 
![20231114_154536](https://github.com/arendst/Tasmota/assets/74438819/35e5f127-2851-4e1b-9621-cb492b2bde12)

Rule1 on tele-DHT11#Temperature do DisplayText [s3p21x0y0y] %value%C endon
i need to move the text down a bit but for the life of me cannot figure out how to do so , any help would be much apreciated.
thanks 
martin

**Comment by sfromis** [comment, marked as answer]
I'd try adjusting the y coordinate, `y0` to a higher number, and drop the extraneous `y` at the end.

**Reply by montyfert** [reply]
thank you
